### DISCUSSION TITLE
App store can not show any app

### DISCUSSION THREAD

**Discussion by vampywiz17** [LingmoOS/Community#12] [Bugs Report/缺陷提交] [answered]
App store not show any app and also not possibe to search here.

![image](https://github.com/user-attachments/assets/f8d2463a-d056-4f83-8235-bb980ddb662c)

**Comment by elysia-best** [comment, marked as answer]
This appstore is known to be broken. @lingmo-dream Consider to remove it or replace it with GNOME's one

**Reply by angrycsm** [reply]
How to remove lingmo store?

**Reply by elysia-best** [reply]
> How to remove lingmo store?

Try `sudo apt remove lingmo-appstore` and install another app store you like~
